src/Controller/Api/FundingOrganizationController.php

```diff
@@ -338,11 +403,16 @@ public function postLogoAction($id, Request $request)
      *   }
      * )
      *
-     * @Rest\Put("/{id}/logo")
+     * @Route(
+     *     "/api/funding-organizations/{id}/logo",
+     *     name="pelagos_api_funding_organizations_put_logo",
+     *     methods={"PUT"},
+     *     defaults={"_format"="json"}
+     *     )
      *
      * @return Response A Response object with an empty body and a "no content" status code.
      */
-    public function putLogoAction($id, Request $request)
+    public function putLogoAction(int $id, Request $request)
     {
         return $this->putProperty(FundingOrganization::class, $id, 'logo', $request);
     }
```

Comment:
I guess we never supported a delete logo action, verified in 5.26.0

Reply:
The logo is broken, don't use it.